### DISCUSSION TITLE
【mod投稿】我汉化的22个mod

### DISCUSSION THREAD

**Discussion by IMGOLDkey** [imgradeone/revolution#10] [Mod 投稿 / Mod Submission]
包括
![image](https://user-images.githubusercontent.com/54227087/142755564-b01c622c-7c14-4ee1-83a5-a345b049a542.png)
![image](https://user-images.githubusercontent.com/54227087/142755568-460600ca-4cb7-4845-ab23-e01be664c441.png)
![image](https://user-images.githubusercontent.com/54227087/142755579-c8b55c1e-bbd1-4b8e-b452-507c5f7ad71a.png)

链接：https://pan.baidu.com/s/14UKR53MEQ2hmwxm0phvHsg 
提取码：GOLD

**Comment by imgradeone** [comment]
感谢您的投稿，但由于您的作品是汉化作品，因此我们将考虑先收录原版作品，再附上您的汉化版链接。

请注意，我们将会 **二次上传** 您的作品，且对于部分内置 DDLC RPA 文件的 Mod 将删除相应文件。

> 同时，若您的作品中包含在 [不会在 Mod 仓库中出现的 Mod
](https://revolution.dokimod.cn/not-included-in-modlib.html) 列表中，很遗憾，您相应 Mod 的汉化作品将无法上架 Mod 仓库。（不影响其他正常 Mod，如果您汉化的 Mod 并没有出现在本列表中，忽略即可）

如果您授权我们使用您的汉化作品，并同意上述要求，请引用并回复本帖（Quote Reply）。

**Reply by IMGOLDkey** [reply]
没问题。另外我在用模板的时候遇到一个问题，对话结束时对话框不会自动消失，我问了另外一个做mod的人（中国老九），他说他也有这种情况，请问可以加一下我的QQ1569274504，帮我看看到底是哪里出问题了吗？（这么久才回复，不好意思呀）

**Reply by imgradeone** [reply]
> 没问题。另外我在用模板的时候遇到一个问题，对话结束时对话框不会自动消失，我问了另外一个做mod的人（中国老九），他说他也有这种情况，请问可以加一下我的QQ1569274504，帮我看看到底是哪里出问题了吗？（这么久才回复，不好意思呀）

```renpy
    window auto
```

使用这个函数即可让对话框自动隐藏。
